src/sentry/mediators/external_requests/alert_rule_action_requester.py

```diff
@@ -74,7 +74,7 @@ def _build_headers(self):
         return {
             "Content-Type": "application/json",
             "Request-ID": request_uuid,
-            "Sentry-App-Signature": self.sentry_app.build_signature(self.body),
```

Comment:
Do you have any idea where this header was from? Was it probably a typo?

Reply:
I couldn't find `Sentry-App-Signature` documented anywhere, but just noticed it as a header on the request while testing out some SentryApp stuff. It definitely looks like it should have followed the `Sentry-Hook-Signature` pattern which is documented.

I'll bring this up to the rest of the team tomorrow just before merging to sanity check but it seems like this is what was originally intended.